<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>944</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>944</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>924</width>
      <height>614</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Carbon Steel Microstructure Analyzer</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="descriptionLabel">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Image processing for characterization of SAE 1020 and SAE 1045 carbon steels.</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="selectImagePushButton">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Select Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="processImagePushButton">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Process Image</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="innerMainHorizontalLayout" stretch="0,0">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="leftImageFrame">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>450</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <widget class="QLabel" name="leftImageLabel">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>460</width>
            <height>450</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>460</width>
            <height>450</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>460</width>
            <height>450</height>
           </size>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>156</red>
                <green>156</green>
                <blue>156</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>156</red>
                <green>156</green>
                <blue>156</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>190</red>
                <green>190</green>
                <blue>190</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>Original Image</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="rightMaskFrame">
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>450</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="rightMaskLabel">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>460</width>
            <height>450</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>460</width>
            <height>450</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>460</width>
            <height>450</height>
           </size>
          </property>
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>156</red>
                <green>156</green>
                <blue>156</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>156</red>
                <green>156</green>
                <blue>156</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>190</red>
                <green>190</green>
                <blue>190</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Ubuntu Mono</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Result Mask</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="innerMainTextHorizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="phasesResultsLabel">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>[ Ferrite: 0.00000% ][ Pearlite: 0.00000% ]</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="saveResultsPushButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Save Results</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadResultsPushButton">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Load Results</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
